pos-ui-extension-smart-grid/shopify.extension.toml.liquid

```diff
@@ -1,10 +1,11 @@
-# The version of APIs your extension will receive. Learn more:
-# https://shopify.dev/docs/api/usage/versioning
+{%- if flavor contains "preact" -%}
+api_version = "2025-10"
+{% else %}
 api_version = "2025-07"
-
+{% endif %}
 [[extensions]]
 type = "ui_extension"
-name = "{{ name }}"
+name = "{%- if flavor contains "preact"-%}remote-dom-{%- endif %}{{ name }}"
```

Comment:
Need to prepend the name with remote-dom per https://github.com/Shopify/extensibility/blob/henry/07-28-fix_parsing_in_admin_api_plugin_of_comments/sandboxes/extensibility-host-mobile-loader/src/runtimes/UiExtensionRuntime.ts#L15